.github/workflows/publish.yml

```diff
@@ -0,0 +1,48 @@
+name: Publish Docker image
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  push_to_registry:
+    name: Push Docker image to GitHub Packages
+    runs-on: ubuntu-latest
+    permissions:
+      packages: write
+      contents: read
+    strategy:
+      matrix:
+        service: [api, ingestion-server, index-worker, analytics]
+        include:
+          - api:
+            context: openverse_api
+            file: Dockerfile
+          - ingestion-server:
+            context: ingestion_server
+            file: Dockerfile
+          - index-worker:
+            context: ingestion_server
+            file: Dockerfile_worker
+          - analytics:
+            context: analytics
+            file: Dockerfile
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+      - name: Log in to GitHub Docker Registry
+        uses: docker/login-action@v1
+        with:
+          registry: https://ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Build and push ${{ matrix.service }} image
```

Comment:
Not sure if we can use the expressions in the name. Very cool if we can, though!

Reply:
We can! This run is the result of doing so: https://github.com/WordPress/openverse-api/runs/4599659216?check_suite_focus=true

Reply:
I learn something new about GitHub on a daily basis. Thanks!